core/commands/generate_product_repositories_command.rb

```diff
@@ -194,6 +191,7 @@ def parse_maxscale_ci(config)
     releases = []
     releases.concat(parse_maxscale_ci_rpm_repository(config['repo']['rpm']))
     releases.concat(parse_maxscale_ci_deb_repository(config['repo']['deb']))
+    releases.concat(parse_maxscale_ci_repository_for_docker)
```

Comment:
Why all my recent changes are reverted??? Please rollback this to the integration branch state.